runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -8973,7 +8978,7 @@ J9::Z::TreeEvaluator::VMgenCoreInstanceofEvaluator(TR::Node * node, TR::CodeGene
             if (comp->getOption(TR_TraceCG))
                traceMsg(comp, "%s: Emitting Class Equality Test\n", node->getOpCode().getName());
             cg->generateDebugCounter(TR::DebugCounter::debugCounterName(comp, "instanceOfStats/(%s)/Equality", comp->signature()),1,TR::DebugCounter::Undetermined);
-             /*   #IF NextTest = GoToFalse
+             /*   #IF NextTest == GoToFalse
               *      branchCond = ifInstanceOf ? (!trueFallThrough ? COND_BE : COND_BNE ) : (init=true ? COND_BE : COND_BNE )
```

Comment:
The condition `init=true` should be `init == true` or, better, just `init`.
I also suggest this would be clearer without the negation of `trueFallThrough`:
```
              *      branchCond = ifInstanceOf ? (trueFallThrough ? COND_BNE : COND_BE) : (init ? COND_BE : COND_BNE)
              *      branchLabel = ifInstanceOf ? (trueFallThrough ? falseLabel : trueLabel) : doneLabel
```
and without the space before each `)`.

Also, `#ENDIF` is missing after line 8986.